griptape/memory/structure/conversation_memory.py

```diff
@@ -16,6 +16,12 @@ class ConversationMemory:
     driver: Optional[BaseConversationMemoryDriver] = field(default=None, kw_only=True)
     runs: list[Run] = field(factory=list, kw_only=True)
     structure: Structure = field(init=False)
+    autoload = field(default=False, kw_only=True)
```

Comment:
@vasinov do you think it should default to `True`? Not sure if having to specify `autoload=True` takes the "auto" out of autoloading 😅

Reply:
I _think_ it makes sense to set to `True` by default.

Reply:
I _think_ it makes sense to set to `True` by default...

And let's add `Bool` to autoload :)